src/addons/ReactFragment.js

```diff
@@ -96,6 +96,15 @@ var ReactFragment = {
   // of its properties.
   create: function(object) {
     if (__DEV__) {
+      if (typeof object !== 'object' || !object) {
+        warning(
+          false,
+          object && typeof object === 'object',
+          'React.addons.createFragment only accepts a single object. Not: %s',
```

Comment:
These arguments look a little mixed up. Shouldn't the format string be the second arg?